paimon-format/pom.xml

```diff
@@ -298,6 +298,10 @@ under the License.
                                     <include>org.apache.parquet:parquet-format-structures</include>
                                     <include>org.apache.parquet:parquet-jackson</include>
                                     <include>commons-pool:commons-pool</include>
+
+                                    <!-- compress -->
+                                    <include>com.github.luben:zstd-jni</include>
+                                    <include>org.xerial.snappy:snappy-java</include>
```

Comment:
The JNI class should not be relocated: https://github.com/luben/zstd-jni#limitations